### PR TITLE
Add and endpoint to feed the foundation's performance chart

### PR DESCRIPTION
**Pull request by gabrielpoca**
The new endpoint returns a (value, timestamp) pair, where the value is the sum of the current total value of a foundation (shares * price per share) plus the amount claimed.

### CODE REVIEW

File: src/jobs/collectFoundationsPerformance.ts [hunk 1]
```diff
@@ -30,20 +31,36 @@ const JOB_NAME = 'collectFoundationsPerformance';
...
-      const metric = repository.create({
-        foundation: foundation.id,
-        amountClaimed: foundation.amountClaimed,
-        amountDeposited: foundation.amountDeposited,
-        shares: foundation.shares,
-      });
-
-      await repository.save(metric);
-    }),
+  const foundaionMetricsRaw = await Promise.all(
```
Comment (0xrin1) [inline src/jobs/collectFoundationsPerformance.ts:36]: typo here

File: src/api/metrics.ts [hunk 2]
```diff
@@ -25,6 +30,47 @@ export default function (
...
+          'foundation_metric.amountClaimed as "amountClaimed"',
+          'v.value as performance',
+          'foundation_metric.createdAt as "createdAt"',
+        ])
+        .where('foundation_metric.id = :id', { id: request.params.id })
+        .where('foundation_metric.createdAt >= :start', { start })
+        .andWhere('foundation_metric.createdAt <= :end', { end })
+        .innerJoin('foundation_metric.vaultMetric', 'v')
+        .limit(MAX_RECORDS)
+        .getRawMany();
```
Comment (0xrin1) [inline src/api/metrics.ts:54]: could be helpful to pack this away into a repository file or something to make more semantic & available for reuse if ever needed

File: src/jobs/collectFoundationsPerformance.ts [hunk 3]
```diff
@@ -30,20 +31,36 @@ const JOB_NAME = 'collectFoundationsPerformance';
...
+
+    await transactionalEntityManager.getRepository(FoundationMetric).save(
+      foundaionMetricsRaw.map((foundation) => ({
+        ...foundation,
+        vaultMetric: vaultMetric,
+      })),
+    );
+  });
+
+  console.log('done');
```
[the remark this answers is not in the document]
Comment (joaop21) [reply src/jobs/collectFoundationsPerformance.ts:65]: I doubt we need to log anything here, probably it was for debugging purposes (I'm assuming that). I think that if we need to log anything and attach it to this specific job, we could use the context that is defined in the `src/jobs/helpers.ts` and passed to every job.

File: src/api/metrics.ts (hunk 2 above)
Comment (gabrielpoca) [reply src/api/metrics.ts:54]: this is a highly specific to this endpoint which feeds the chart :/

File: src/jobs/collectFoundationsPerformance.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (gabrielpoca) [reply src/jobs/collectFoundationsPerformance.ts:65]: ahahah sorry, I forgot to clean-up!

File: src/api/metrics.ts (hunk 2 above)
Comment (joaop21) [reply src/api/metrics.ts:54]: Probably not in this PR, but somewhere in the near future, I agree that we need to move these queries to a logical layer.

**Review by 0xrin1 — APPROVED** [review]

**Pull request merged**